src/layers/domain/repository/device_repository/v2.py

```diff
@@ -49,33 +90,31 @@ def _dynamodb_update_expression(updated_fields: dict):
     )
 
 
-def _device_primary_keys(
-    device_id: str, device_keys: list[DeviceKey], device_tags: list[DeviceTag]
-) -> list[dict]:
+def _device_root_primary_key(device_id: str) -> dict:
+    """
+    Generates one fully marshalled (i.e. {"pk": {"S": "123"}} DynamoDB
+    primary key (i.e. pk + sk) for the provided Device, indexed by the Device ID
+    """
+    root_pk = TableKey.DEVICE.key(device_id)
+    return marshall(pk=root_pk, sk=root_pk)
+
+
+def _device_non_root_primary_keys(device_keys: list[DeviceKey]) -> list[dict]:
     """
     Generates all the fully marshalled (i.e. {"pk": {"S": "123"}} DynamoDB
     primary keys (i.e. pk + sk) for the provided Device. This is one primary key
-    plus an additional primary key for every value of Device.keys
+    for every value of Device.keys and Device.tags
```

Comment:
not tags

Reply:
Has `Device.tags` been removed here? Maybe the change was undone with some of the rebases?